src/v/redpanda/application.cc

```diff
@@ -1952,6 +1951,11 @@ void application::start_bootstrap_services() {
 void application::wire_up_and_start(::stop_signal& app_signal, bool test_mode) {
     // Setup the app level abort service
     construct_service(_as).get();
+    construct_service(stress_fiber_manager).get();
+    _deferred.emplace_back([this] {
```

Comment:
Shouldn't sharded already do this when `stop` is called (which is deferred in `construct_service`?

See: https://github.com/redpanda-data/seastar/blob/245e0ccfa6d58d7e0dca2b4034ce1bc43e39bdc5/include/seastar/core/sharded.hh#L193-L197

Reply:
IIUC that's referring to `sharded<Service>::stop()`, which just destructs the underlying `Service` on each core. We also need to call `stress_fiber_manager::stop()` to call stop the fibers.

Reply:
> which just destructs the underlying Service on each core. 

I believe it **also** calls `stop` for each sharded service if that function exists:
https://github.com/redpanda-data/seastar/blob/245e0ccfa6d58d7e0dca2b4034ce1bc43e39bdc5/include/seastar/core/sharded.hh#L722C17-L722C17
https://github.com/redpanda-data/seastar/blob/245e0ccfa6d58d7e0dca2b4034ce1bc43e39bdc5/include/seastar/core/sharded.hh#L688-L708

Reply:
yeh, stop is optional. but if it's present, it will be invoked (unlike sharded::start which just does the constructor part)

Reply:
Oh wow that's cool! Thanks for the callout.